server/src/main/java/com/paypal/selion/proxy/SeLionRemoteProxy.java

```diff
@@ -444,22 +451,35 @@ public synchronized void forceNodeShutdown() {
         HttpResponse response = sendToNodeServlet(NodeForceRestartServlet.class, nvps);
         if (response == null) {
             proxyLogger.warning("Node " + getId() + " failed to shutdown and returned a null response.");
-            LOGGER.exiting(false);
+            // stop the polling thread, mark this proxy as down, force this proxy to re-register
+            teardown("it failed to shutdown and must re-register");
```

Comment:
wasn't sure which was a better approach here.. 
1. This (as I have done here)
2. Increase `uniqueSessionCount ` by `uniqueSessionCount + uniqueSessionCount ` and let this proxy continue
3. Track the failure as a new member variable which can be queried or displayed .. something like `hasErrors`
4. 2 & 3
5. Other....